frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -56,15 +75,43 @@ export const BrowseModels = ({
 
   if (modelsFiltered.length) {
     return (
-      <GridContainer role="grid">
-        {groupsOfModels.map(groupOfModels => (
-          <ModelGroup
-            models={groupOfModels}
-            key={`modelgroup-${groupOfModels[0].collection.id}`}
-            localeCode={localeCode}
-          />
-        ))}
-      </GridContainer>
+      <>
+        {shouldShowBanner && (
+          <Paper
+            mt="1rem"
+            p="1rem"
+            color="text-dark"
+            bg="brand-lighter"
+            shadow="0"
+            radius="0.25rem"
+            role="complementary"
+          >
+            <Flex>
+              <BannerModelIcon name="model" />
+              <Text size="md" lh="1rem" mr="1rem">
+                {t`Models help curate data to make it easier to find answers to questions all in one place.`}
+              </Text>
+              <BannerCloseButton
+                onClick={() => {
+                  dismissBanner();
+                }}
+              >
+                <Icon name="close" />
+              </BannerCloseButton>
+            </Flex>
+          </Paper>
+        )}
```

Comment:
this would be a lot cleaner if it was its own component, it could probably keep track of its own state too.

Reply:
Its state has style implications for the grid, unfortunately, so I wouldn't be able to move the state into a smaller component

Reply:
I think I successfully handled this with a bit of negative margin. take a look:

[`78cb0ba` (#38635)](https://github.com/metabase/metabase/pull/38635/commits/78cb0bada41c1f878a55e4373896c43e046912e8#r1484872753)